tests/testthat/test-error.R

```diff
@@ -39,10 +39,25 @@ test_that("returns the correct linting", {
     equals_na_linter
   )
 
+  # Error message changed in R-devel as of 2020/12
+  old_lang <- Sys.getenv("LANG")
+  Sys.setenv(LANG = "en")
```

Comment:
I believe the variable is `LANGUAGE` btw

Reply:
Thanks, had a brain fart :D
Tests were failing locally with LANG.